plugin/web/components/sidebar/Layout/LayoutHeader.tsx

```diff
@@ -1,11 +1,11 @@
 import { CloseOutlined } from "@ant-design/icons";
+import { styled } from "@web/theme";
 import { Button, Menu, MenuProps, Row } from "antd";
 import { Header } from "antd/lib/layout/layout";
 import { MenuInfo } from "rc-menu/lib/interface";
 import React, { memo } from "react";
 
-import { styled } from "../../../theme";
-import { PlateauLogo } from "../../common/Icon/icons";
+import Icon from "../../common/Icon";
```

Comment:
Please use the @web alias you created now

Reply:
done

Reply:
[pr-fix1](https://github.com/eukarya-inc/reearth-plateauview/pull/30/commits/4368dbe8b4bb396983a7cc21a87dc8497771a482)